torch/quantization/quantize_fx.py

```diff
@@ -392,6 +395,11 @@ def prepare_fx(
       with a similar structure as qconfig_dict except it will contain
       configurations specific to equalization techniques such as input-weight
       equalization.
+      `backend_config_dict`: a dictionary that specifies how operators are quantized
```

Comment:
Can we update this to what is supported currently? Right now, this PR is just introducing the argument, should we have some functionality with this dict before adding it?